### PR TITLE
Storage get count

### PR DESCRIPTION
**Pull request by yuan-fang-228**
add get() and count() methods to db_storage.py and file_storage.py

### CODE REVIEW

File: models/engine/file_storage.py [hunk 1]
```diff
@@ -55,7 +74,7 @@ def reload(self):
                 jo = json.load(f)
             for key in jo:
                 self.__objects[key] = classes[jo[key]["__class__"]](**jo[key])
-        except:
+        except Exception:
```
Comment (cienyanhi) [inline models/engine/file_storage.py:77]: Well done on adding 'Exception' to except to meet PEP8 pycodetyle!

File: models/engine/db_storage.py [hunk 2]
```diff
@@ -40,6 +40,24 @@ def __init__(self):
...
+                return obj
+        return None
+
+    def count(self, cls=None):
+        """return number of objects in storage of the given class"""
+        classList = classes.values()
+        if cls is not None:
+            classList = [cls]
+        count = 0
+        for clss in classList:
```
Comment (cienyanhi) [inline models/engine/db_storage.py]: Unclear variable name. What about 'each_class'?

File: models/engine/file_storage.py [hunk 3]
```diff
@@ -34,6 +34,25 @@ def all(self, cls=None):
...
+                    return value
+        return None
+
+    def count(self, cls=None):
+        """return the number of objects in storage of the given class"""
+        classList = classes.values()
+        if cls is not None:
+            classList = [cls]
+        count = 0
+        for clss in classList:
```
Comment (cienyanhi) [inline models/engine/file_storage.py]: Unclear variable name. What about 'each_class'?

**Comment by cienyanhi** [comment]
Good job the code is working!! Correct method names were used, easy to read, appropriate length and all the docstrings documentations are available. 
My suggestion is to change 'clss' variable name to 'each_class' to make it clearer when other people go back to look at it. I also suggest to add more tests to test edge cases.

**Review by cienyanhi — CHANGES_REQUESTED** [review]

**Review by cienyanhi — APPROVED** [review]

**Comment by cienyanhi** [comment]
Thanks for changing the variable name quickly! Happy to merge it back to main branch now!
